lib/utils/Utils.dart

```diff
@@ -1197,4 +1198,33 @@ class Utils {
     if (placemarks.isEmpty) { return null; }
     return placemarks.first.locality;
   }
+
+  static void requestInAppReview() async {
+    var myReports = await ApiSingleton().getReportsList();
+    var numReports = myReports.length;
+
+    if (numReports < 3) { return; }
+
+    var shouldRequestReview = false;
```

Comment:
Maybe using one-line?

```dart
var shouldRequestReview = (numReports == 3 || numReports == 4 || numReports >= lastReportCount + 3 || now - lastReviewRequest >= 14 * 24 * 60 * 60 * 1000);
```

Reply:
seems good to me, let's keep this one-line